content/comm/blog/index.en.md

```diff
@@ -1,211 +0,0 @@
----
-title: "R-Ladies Blog"
```

Comment:
do we need to add a redirect or is it ok since the website has search and a 404 page?

Reply:
then in `netlify.toml`? https://docs.netlify.com/routing/redirects/#syntax-for-the-netlify-configuration-file

Reply:
I made an alias in stead, I think where appropriate that using hugo for that makes more sense?